test/controllers/drivers_controller_test.rb

```diff
@@ -108,14 +145,30 @@
       # Ensure there is an existing driver saved
       # Assign the existing driver's id to a local variable
       # Set up the form data
+      driver = Driver.create(name: "Mair Bear", vin: "123ABC456EFG789HI", available: true)
+      driver_hash = {
+        driver: {
+          name: "changed driver:(",
+          vin: "12345678910ABCDEF",
+          available: true,
+        },
+      }
 
       # Act-Assert
       # Ensure that there is no change in Driver.count
+      expect {
+        patch driver_path(driver.id), params: driver_hash
+      }.wont_change "Driver.count"
 
       # Assert
       # Use the local variable of an existing driver's id to find the driver again, and check that its attributes are updated
       # Check that the controller redirected the user
+      expect(Driver.last.name).must_equal driver_hash[:driver][:name]
+      expect(Driver.last.vin).must_equal driver_hash[:driver][:vin]
+      expect(Driver.last.available).must_equal driver_hash[:driver][:available]
```

Comment:
This works for this, no problem.  However other databases might not put the edited item last.  Instead it might be a good habit to find the driver again and then verify that things are changed.